### PR TITLE
fix: the editor cannot apply empty string attributes to wysiwyg node

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* fixed that the editor cannot apply empty string attributes to wysiwyg node
* related issue(#1710)
* TODO: should add DOMPurify for proper sanitizing

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: apps/editor/src/__test__/unit/editor.spec.ts [hunk 1]
```diff
@@ -864,6 +864,25 @@ describe('editor', () => {
 
         expect(wwEditor.innerHTML).toContain(result);
       });
+
+      it('should keep the html block attributes with an empty string after changing the mode', () => {
```
Comment (dongsik-yoo) [inline apps/editor/src/__test__/unit/editor.spec.ts:868]: 설명에 block이 들어갈 필요는 없는 거죠?

**Review by dongsik-yoo — COMMENTED** [review]
리뷰 완료합니다. 수고하셨습니다.

**Review by adhrinae — APPROVED** [review]
리뷰 완료합니다.

File: apps/editor/src/__test__/unit/editor.spec.ts (hunk 1 above)
Comment (adhrinae) [reply apps/editor/src/__test__/unit/editor.spec.ts:868]: 제가 보기에도 굳이 언급 안해도 될 것 같은 단어네요.

File: apps/editor/src/__test__/unit/editor.spec.ts [hunk 2]
```diff
@@ -864,6 +864,25 @@ describe('editor', () => {
...
+          previewHighlight: false,
+          // add iframe html block renderer
+          customHTMLRenderer: createHTMLrenderer(),
+        });
+
+        editor.changeMode('wysiwyg');
+
+        const result = oneLineTrim`
+          <iframe width="" height="" src="" class="html-block"></iframe>
+          <p>test</p>
```
[the remark this answers is not in the document]
Comment (js87zz) [reply apps/editor/src/__test__/unit/editor.spec.ts:881]: 딱히 의미가 없네요. 해당 텍스트는 제거하겠습니다~

**Pull request merged**